packages/mdc-checkbox/_mixins.scss

```diff
@@ -156,7 +156,6 @@
   &-checked-unchecked,
   &-indeterminate-unchecked {
     .mdc-checkbox__background {
-      animation-duration: $mdc-checkbox-transition-duration * 2;
```

Comment:
this changes seems to have sneaked into this PR...can you make sure packages/mdc-checkbox/_mixins.scss is the exact same as our master branch?